pkg/event/stats.go

```diff
@@ -100,6 +104,32 @@ func (d *Data) AddEvent(event EventChannel) {
 	d.logData = details.logData
 	d.Details = append(d.Details, details)
 	if len(StateRegisterer.Subscribers) > 0 {
+		glog.Infof("notify state changed for %s", event.ProcessName)
 		go StateRegisterer.notify(event.ProcessName, event.State)
 	}
 }
+
+// ToString ... data
+func (d *Data) toString() string {
+	out := strings.Builder{}
+	out.WriteString("  logData : " + d.logData)
+	out.WriteString("  state: " + string(d.State))
+	out.WriteString("   process name: " + string(d.ProcessName))
+	out.WriteString("  Data Details: " + d.Details.toString())
+	out.WriteString("-----\r\n")
+	return out.String()
+}
+
+// toString ... data details
+func (dd DDetails) toString() string {
+	out := strings.Builder{}
+	for _, d := range dd {
+		out.WriteString("  Iface name: " + d.IFace)
+		out.WriteString("  state: " + string(d.State))
+		out.WriteString("  clock type: " + string(d.ClockType))
+		out.WriteString("  process name: " + strconv.FormatBool(d.sourceLost))
```

Comment:
out.WriteString(" signal source: " + string(d.signalSource))
out.WriteString(" source lost: " + strconv.FormatBool(d.sourceLost))

Reply:
Done

Reply:
I don't see it

Reply:
click on  view view changes